docs/modules/druid/pages/index.adoc

```diff
@@ -1,6 +1,49 @@
 = Stackable Operator for Apache Druid
+:description: The Stackable Operator for Apache Druid is a Kubernetes operator that can manage Apache Druid clusters. Learn about its features, resources, dependencies, and demos, and see the list of supported Druid versions.
+:keywords: Stackable Operator, Apache Druid, Kubernetes, operator, DevOps, engineer, CRD, StatefulSet, ConfigMap, Service, ZooKeeper, HDFS, S3, Kafka, Trino, OPA, demo, version
+
+The Stackable Operator for Apache Druid is a Kubernetes operator that can manage https://druid.apache.org/[Apache Druid] clusters. With this operator you can deploy and manage Druid clusters on Kubernetes. This operator provides several resources and features to manage Druid clusters efficiently.
+
+== Getting Started
+
+To get started with the Stackable Operator for Apache Druid, follow the xref:druid:getting_started/index.adoc[Getting Started guide]. The Operator is installed along with the _DruidCluster_ CustomResourceDefinition, which supports five xref:home:concepts:roles-and-role-groups.adoc[roles]: **Router**, **Coordinator**, **Broker**, **MiddleManager** and **Historical**. These roles correspond to https://druid.apache.org/docs/latest/design/processes.html[Druid processes].
```

Comment:
I'm just making it in italics because I'm introducing the term here, in general there should be no special formatting for the terms. That's how the offical k8s docs do it too, and we're just doing it like them.

Reply:
Just fyi, we have a style guide: https://docs.stackable.tech/home/stable/contributor/style_guide.html

And in there the k8s style guide is referenced: https://kubernetes.io/docs/contribute/style/style-guide/#use-upper-camel-case-for-api-objects